packages/api-client/src/v2/features/app/hooks/use-app-sidebar.ts

```diff
@@ -247,8 +246,19 @@ export const useSidebarState = ({
 
   /** Keep the router and the sidebar state in sync */
   watch(
-    [() => toValue(workspaceStore), documentSlug, path, method, exampleName],
-    ([_newWorkspace, newDocument, newPath, newMethod, newExample]) => {
+    [
+      () => toValue(workspaceStore),
+      () => toValue(documentSlug),
+      () => toValue(path),
+      () => toValue(method),
+      () => toValue(exampleName),
+    ],
+    () => {
+      const newDocument = toValue(documentSlug)
+      const newPath = toValue(path)
+      const newMethod = toValue(method)
+      const newExample = toValue(exampleName)
```

Comment:
so if they are already reactive, just watch them directly
```suggestion
    [
      workspaceStore,
      documentSlug,
      path,
      method,
      exampleName,
    ],
    () => {
      const newDocument = toValue(documentSlug)
      const newPath = toValue(path)
      const newMethod = toValue(method)
      const newExample = toValue(exampleName)
```
OR if you want to to value them before you don't need to do it later

```suggestion
    [
      () => toValue(workspaceStore),
      () => toValue(documentSlug),
      () => toValue(path),
      () => toValue(method),
      () => toValue(exampleName),
    ],
    ([newDocument, newPath, newMethod, newExample]) => {
```